spec/openapi.yaml

```diff
@@ -6468,6 +6468,84 @@ components:
             The subject field is used to denote subject of the MMS message and
             has a maximum size of 64 characters long
           example: New Products
+    DetailMessage:
+      title: Message
+      required:
+        - content
+        - destination_number
+      type: object
+      properties:
+        content:
+          maxLength: 5000
+          minLength: 1
+          type: string
+          description: Content of the message
+          example: Hello world!
+        destination_address:
+          maxLength: 15
+          minLength: 1
+          type: string
+          description: Destination number of the message
+          example: '+61491570156'
+        destination_address_country:
+          type: string
+          description: Country of the destination address
+          example: 'AU'
+        format:
+          $ref: '#/components/schemas/Format'
+        metadata:
+          type: object
+          description: >-
+            Metadata for the message specified as a set of key value pairs, each
+            key can be up to 100 characters long and each value can be up to 256
+            characters long
+
+            ```
+
+            {
+               "myKey": "myValue",
+               "anotherKey": "anotherValue"
+            }
+
+            ```
+          example:
+            - "myKey": "myValue"
+            - "anotherKey": "anotherValue"
+        source_address:
+          type: string
+          example: '+61491570156'
```

Comment:
Keep contacts clearly fictitious. That example look like the reporter's real phone number. Suggested: `+61490246135` or `+6149123654`